easybuild/easyconfigs/m/metaerg/metaerg-1.2.3-intel-2019b-Python-2.7.16.eb

```diff
@@ -0,0 +1,63 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+#
+# NOTE: Operating MetaErg requires reference DB, see:
+# https://github.com/xiaoli-dong/metaerg#metaerg-reference-db
+
+easyblock = 'Tarball'
+
+name = 'metaerg'
+version = '1.2.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/xiaoli-dong/metaerg'
+description = """MetaErg is a stand-alone and fully automated metagenomic and metaproteomic data annotation pipeline."""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+
+github_account = 'xiaoli-dong'
+source_urls = [GITHUB_SOURCE]
+sources = ['%(version)s.tar.gz']
+checksums = ['dbe8ebc29ff8464d00f8a90f4dd9e48eaaadb3e6a95cbf7c620cac4f9d6c8c3c']
+
+dependencies = [
+    ('Python', '2.7.16'),
+    ('Perl', '5.30.0'),
+    ('BioPerl', '1.7.2'),
+    ('BLAST+', '2.9.0'),
+    ('ARAGORN', '1.2.38'),
+    ('DIAMOND', '0.9.30'),
+    ('HMMER', '3.2.1'),
+    ('MinPath', '1.4', versionsuffix),
+    ('prodigal', '2.6.3'),
+    ('MinCED', '0.4.2', '-Java-11'),
+    ('Bio-EUtilities', '1.76'),
+    ('LWP-Protocol-https', '6.07'),
```

Comment:
@Darkless012 This dependency should be removed, is now included with `Perl` via #10412?